start_esp/nginx-auto.conf.template

```diff
@@ -113,6 +113,24 @@ http {
 % endif
 
 % if location.proto == 'grpc':
+  % if cors_preset == 'basic':
```

Comment:
I will like to make this cors support not grpc specific?  it can also support http too.

Reply:
Yes, I'll move this block out of grpc when I get the time.